seed/views/00501_team_capacity_breakdown_view.js

```diff
@@ -1,12 +1,13 @@
 exports.seed = function (knex, Promise) {
-  var sql = `CREATE VIEW app.team_capacity_breakdown_view
+  var view = `CREATE VIEW app.team_capacity_breakdown_view
         WITH SCHEMABINDING
         AS
         SELECT
             t.id AS id
           , wo.id AS link_id 
-          , CONCAT(om.forename, ' ', om.surname) AS name
-          , omt.grade_code as grade_code
+          , om.forename
+          , om.surname
```

Comment:
Going to be an update to `get-capacity-breakdown` in the web to concatenate these?

Reply:
yup, working on that now.

Reply:
Covered by https://github.com/ministryofjustice/wmt-web/pull/175